pkg/virt-api/api.go

```diff
@@ -77,13 +77,13 @@ const (
 	// Default address that virt-api listens on.
 	defaultHost = "0.0.0.0"
 
-	defaultConsoleServerPort = 8186
+	DefaultConsoleServerPort = 8186
 
 	defaultCAConfigMapName     = "kubevirt-ca"
 	defaultTlsCertFilePath     = "/etc/virt-api/certificates/tls.crt"
 	defaultTlsKeyFilePath      = "/etc/virt-api/certificates/tls.key"
-	defaultHandlerCertFilePath = "/etc/virt-handler/clientcertificates/tls.crt"
-	defaultHandlerKeyFilePath  = "/etc/virt-handler/clientcertificates/tls.key"
+	DefaultHandlerCertFilePath = "/etc/virt-handler/clientcertificates/tls.crt"
+	DefaultHandlerKeyFilePath  = "/etc/virt-handler/clientcertificates/tls.key"
```

Comment:
Perhaps changing the visibility scope of those constants is not needed anymore.

Reply:
Absolutely!